src/bvals/comms/bnd_info.hpp

```diff
@@ -78,11 +71,68 @@ struct BndInfo {
                                  CommBuffer<buf_pool_t<Real>::owner_t> *buf);
 };
 
+struct ProResInfo {
+  int ntopological_elements = 1;
+  // Has to be large enough to allow for maximum integer
+  // conversion of TopologicalElements
+  SpatiallyMaskedIndexer6D idxer[10];
+
+  CoordinateDirection dir;
```

Comment:
Is this just `+1` or `-1`? Or is it `x`, `y`, `z`? Presumably more information is needed to specify buffer?

Reply:
hm, I'm not sure why that is there. It may have just been copy and pasted along from the original `BndInfo` implementation...